webapp/src/components/expanded_view/component.tsx

```diff
@@ -561,6 +604,34 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                                 </button>
                             </OverlayTrigger>
 
+                            <div style={{position: 'relative'}}>
+                                {this.renderEmojiPicker()}
+                                <OverlayTrigger
+                                    key='tooltip-emoji-picker'
+                                    placement='top'
+                                    overlay={
+                                        <Tooltip
+                                            id='tooltip-emoji-picker'
+                                        >
+                                            <span>{'Make a reaction'}</span>
```

Comment:
In Channels this is `Add Reaction`, but 0/5
![image](https://user-images.githubusercontent.com/3924815/191786780-5026eb11-cbec-4a55-a1a0-00bb856237a9.png)